src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -148,7 +148,11 @@ public function getRouteCollectionForRequest(Request $request)
                 return $collection;
             }
 
-            if (preg_match('/\/$/', $resourceLocator) && $prefix) {
```

Comment:
Any reason you didn't just change the check to the following lines:

```php
if (preg_match('/\/$/', $resourceLocator)) {
    // ...
}
```

This works for me as well, without an additional if.

Reply:
Ok, it does not work for the homepage :see_no_evil: What about the following approach:

```php
$prefix = $attributes->getAttribute('resourceLocatorPrefix') ?? '';

// ...

if (preg_match('/\/$/', $resourceLocator) && $resourceLocator !== '/) {
    // ...
}
```

This would avoid the deeper nested if and the need to change a variable in the middle of the function, which could lead to bad surprises.

Reply:
@danrot this will not work for homepage with prefix e.g. `/de/`:

Most cases are tested and `if (preg_match('/\/$/', $resourceLocator) && $prefix) {` was in our testing the only if statement which does really work for all cases.

Reply:
Ok, sounds reasonable.